CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.86.1
+
+- Packit _0.86.0_ was not released due to an internal bug; this version brings to the users the _0.86.0_ features
```

Comment:
It has been released though, just not on PyPI.

Reply:
I can specify "not released on Pypi and Fedora"?

Reply:
I'd say something like:

```suggestion
- Packit _0.86.0_ was not released on PyPI due to an internal bug, it should be fixed in this release.
```